doc/Command-Reference.md

```diff
@@ -9285,6 +9318,115 @@ This command is to add or delete a member port into the already created vlan.
   ```
 
+**config vlan member add/del -m -e <vlan_ids> <member_portname>**
+
+This command is to add or delete multiple vlan members into port.
+
+_NOTE: This command is only for adding tagged vlan members into port._
+
+- Usage:
+
+  ```
+  config vlan member add/del [-m|--multiple] [-e|--except_flag] <vlan_ids> <member_portname>
+  ```
+
+- Example (Add the VLAN "Vlan100,Vlan200,Vlan300" in one command on port if they don't exist as members):
+
+  ```
+  admin@sonic:~$ sudo config vlan member add -m  100,200,300 Ethernet0
+  This command will add Ethernet0 as member of the vlan 100, vlan 200, vlan 300
+
+  admin@sonic:~$ sudo config vlan member add -m 100,200,300 Ethernet4
+  This command will add Ethernet4 as member of the vlan 100, vlan 200, vlan 300
+  ```
+
+- Example (Add all VLANs except "Vlan100,Vlan300" from existing VLAN on port if they don't exist as members):
+
+  _Suppose if Vlan 100, Vlan 200, Vlan 300, Vlan 400, Vlan 500, Vlan 700 are existing Vlans._
+
+  ```
+  admin@sonic:~$ sudo config vlan member add -e 100,300 Ethernet4
+  This command will add Ethernet4 as member of Vlan 200, Vlan 400, Vlan 500, Vlan 700
+  ```
+
+**config vlan member add/del range <vlan_id_1> <vland_id_2> <member_portname>**
+
+This command is to add or delete range of vlan members into port.
+
+_NOTE: This command is only for adding tagged vlan members into port._
+
+- Usage:
+
+  ```
+  config vlan member add/del range <vlan_id_1> <vland_id_2> <member_portname>
+  ```
+
+- Example (Add VLAN "Vlan100, Vlan200, Vlan300, Vlan400 ,Vlan500" in one command on port if they don't exist as members):
+
+  ```
+  admin@sonic:~$ sudo config vlan member add range 100 105 Ethernet0
+  This command will add Ethernet0 as member of the Vlan100, Vlan101, Vlan102, Vlan103, Vlan104, Vlan105
+  ```
+
+**config vlan member add/del all <member_portname>**
+
+This command is to add or delete all existing vlan members onto/from port.
+
+_This will not work for untagged vlans._
+
+- Example (Add all VLANs except "Vlan100,Vlan300" from existing VLAN on port if they don't exist as members):
+
+  _Suppose if Vlan 100, Vlan 200, Vlan 300, Vlan 400, Vlan 500, Vlan 700 are existing Vlans._
+
+  ```
+  admin@sonic:~$ sudo config vlan member add all Ethernet4
+  This command will add Ethernet4 as member of Vlan 100, Vlan 200, Vlan 300, Vlan 400, Vlan 500, Vlan 700
+  ```
+
+**config switchport mode trunk|access|routed <member_portname>**
+
+This command is to use to add port modes
+
+- Usage:
+
+  ```
+  config switchport mode trunk <member_portname>
+  ```
+
+- Example:
+
+  ```
+  admin@sonic:~$ sudo config switchport mode trunk Ethernet0
+  This command will add Ethernet0 as member of Default Vlan1 as trunk port
+  ```
+
+- Usage:
+
+  ```
+  config switchport mode access <member_portname>
+  ```
+
+- Example:
+
+  ```
+  admin@sonic:~$ sudo config switchport mode access Ethernet0
+  This command will add Ethernet0 as member of Default Vlan1 as access port

```

Comment:
Why cant we use the command "config vlan member add -u 1 Ethernet0" for the above?